### ISSUE TITLE
No Valid Connection ID found, Volley Error

### ISSUE THREAD

**Issue by Havanna22** [cmtjk/LinkUpConnect#17]
As of several hours, LinkUp Connect does not work, does not connect. See image
![Screenshot_20241210_222229_LinkUpConnect](https://github.com/user-attachments/assets/284f1089-4440-44ec-a796-f0ed082a9213)

**Comment by andrenicGH** [comment]
I can confirm the same issue

**Comment by cmtjk** [comment]
> I can confirm the same issue
> Some additional information can be found in this thread for the xDrip+ app:
> https://github.com/NightscoutFoundation/xDrip/discussions/3808

Perfect, I'll check on this.

**Comment by cmtjk** [comment]
https://github.com/cmtjk/LinkUpConnect/releases/tag/v1.3.5

**Comment by andrenicGH** [comment]
Thanks for quick update!
But I still get a ClientError message, as shown below. I confirm update to 1.3.5, followed by phone restart.
Also updated LibreLinkUp to 4.12.0 yesterday. I can open it and read values OK there.
![image](https://github.com/user-attachments/assets/9514c6d2-6118-4fba-b534-4f5e2115e412)

**Comment by Havanna22** [comment]
It's working for me now. I deinstalled and installed the new version and did Reset.
I get values as I should in LinkUpConnect with the fix

**Comment by cmtjk** [comment]
> Thanks for quick update!
> But I still get a ClientError message, as shown below. I confirm update to 1.3.5, followed by phone restart.
> Also updated LibreLinkUp to 4.12.0 yesterday. I can open it and read values OK there.
> ![image](https://github.com/user-attachments/assets/9514c6d2-6118-4fba-b534-4f5e2115e412)
> 

Please reset or reinstall. It still uses incomplete credentials which were stored previously.

**Comment by andrenicGH** [comment]
> > Thanks for quick update!
> > But I still get a ClientError message, as shown below. I confirm update to 1.3.5, followed by phone restart.
> > Also updated LibreLinkUp to 4.12.0 yesterday. I can open it and read values OK there.
> > ![image](https://private-user-images.githubusercontent.com/136715704/394734606-9514c6d2-6118-4fba-b534-4f5e2115e412.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzM5MjI3MTksIm5iZiI6MTczMzkyMjQxOSwicGF0aCI6Ii8xMzY3MTU3MDQvMzk0NzM0NjA2LTk1MTRjNmQyLTYxMTgtNGZiYS1iNTM0LTRmNWUyMTE1ZTQxMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjExJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIxMVQxMzA2NTlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zMTk1YmRhYTZjNjVkNTQ4YTdlM2RkN2M2YzFkMzcyYWE5YWJhMjYxMDYwOGNkZDcxMTkyMDA3Yjk1MGFjYTQwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.iNsjszJDUk5K_0C5yHPPGiLmVxarwjdbU6CLzmG2ciM)
> 
> Please reset or reinstall. It still uses incomplete credentials which were stored previously.

![image](https://github.com/user-attachments/assets/0dd587fc-a0d5-4755-87c4-721224cfe26a)

I attempted a reset, and also an uninstall-reinstall of the app.
I also reinstalled app, deleted the cache and data buffers, and same message.
Very strange since the issue was resolved for https://github.com/Havanna22

After reset I enter the same email and pw as before. I also have to change the API address from api-de to api-eu.
In new screenshot above I make one attempt with the api-de and get the expected result. But with api-eu it fails as before. 
LibreLinkUp app works and is version 4.12.
Any ideas?

**Comment by andrenicGH** [comment]
Made another attempt this morning, installing the new app on my partner's phone. She has not had the app installed previously. She got exactly the same error as shown above. I have verified that the credentials are correct, as I get "NotAuthenticated" error if I change the password in LinkUpConnect, and I get the region error if I connect to the api-de with the correct password.

I now tried creating a new LibreView account, and got it working. I am having some difficulties getting back to using my original account, but this is clearly not related to the app. So it seems something has bugged out with my account, and the fixed app is indeed working.

Thanks again for your help!
